client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import classNames from 'classnames';
+import { dateI18n } from '@wordpress/date';
+import { getHistory } from '@woocommerce/navigation';
+import { __, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, CardFooter, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import wcpayTracks from 'tracks';
+import type { Dispute } from 'wcpay/types/disputes';
+import { getAdminUrl } from 'wcpay/utils';
+import { formatExplicitCurrency } from 'wcpay/utils/currency';
+import { getDisputeFee, isInquiry } from 'wcpay/disputes/utils';
+import './style.scss';
+
+const formatUnixTimestamp = ( date: string, format: string ) =>
+	dateI18n( format, moment.unix( parseInt( date, 10 ) ).toISOString() );
+
+interface DisputeFooterProps {
+	dispute: Dispute;
+}
+
+const DisputeFooter: React.FC< DisputeFooterProps > = ( { dispute } ) => {
+	const isSubmitted = !! dispute.metadata.__evidence_submitted_at;
+	const isAccepted = dispute.metadata.__closed_by_merchant === '1';
+	const closedDateFormatted = dispute.metadata.__dispute_closed_at
+		? formatUnixTimestamp( dispute.metadata.__dispute_closed_at, 'M j, Y' )
+		: '-';
+
+	const disputeFee = getDisputeFee( dispute );
+	const disputeFeeFormatted = disputeFee
+		? formatExplicitCurrency( disputeFee.fee, disputeFee.currency )
+		: '-';
+	const disputeDocsLinkElement = (
+		// eslint-disable-next-line jsx-a11y/anchor-has-content
```

Comment:
Can we do this inline (below) to avoid the need to `eslint-disable`? Otherwise add a comment saying why it's ok (content is interpolated later). 

(I prefer the former 😁 )

Reply:
Ah I see, it's used multiple times. So just need a comment explaining why the ignore is ok (unless you can think of a way to avoid ignore).

Reply:
Though I have no objection to expanding all uses inline if you prefer that.

Reply:
💯 Getting my head around this one again … comment is the only way. This rule needs to be disabled. Rainy day question: can we do that in WCPay linter config?

Reply:
Another way to solve it – there's an open issue and PR for this at https://github.com/WordPress/gutenberg/issues/21441